app/models/manageiq/providers/amazon/agent_coordinator_worker/runner/response_thread.rb

```diff
@@ -0,0 +1,187 @@
+module ManageIQ::Providers::Amazon::AgentCoordinatorWorker::Runner::ResponseThread
+  require 'xml/xml_utils'
```

Comment:
Is this the one from manageiq-gems-pending?  If so, this should say `'util/xml/xml_utils'` for consistency  (and so we can find it later to delete it)

Reply:
Fixed in https://github.com/ManageIQ/manageiq-providers-amazon/pull/360